python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -807,6 +835,39 @@ def _removeBadSources(self, diaSources):
         self.log.info("Removed %d unphysical sources.", nBadTotal)
         return diaSources[selector].copy(deep=True)
 
+    def _find_trailed_glints(self, diaSources):
+        """Define a new flag column for diaSources that are in a glint trail.
+
+        Parameters
+        ----------
+        diaSources : `lsst.afw.table.SourceCatalog`
+            The catalog of detected sources.
+
+        Returns
+        -------
+        diaSources : `lsst.afw.table.SourceCatalog`
+            The updated catalog of detected sources, with a new bool column
+            called 'trailed_glint' added.
+
+        trail_parameters : `dict`
+            Parameters of all the trails that were found.
+        """
+        trailed_glints = self.findGlints.run(diaSources)
```

Comment:
```suggestion
        glint_trails = self.findGlints.run(diaSources)
```

Reply:
This is the new Butler dataset, not the new diaSource column, so I am leaving the name as-is.